### PR TITLE
ducc is a faggot!!

### PR DESCRIPTION
**Pull request by Dot-Rar**

### CODE REVIEW

**Comment by Dot-Rar** [comment]
Did this one: https://owo.is-pretty.cool/7538c3.png

File: src/main/java/ovh/not/javamusicbot/command/PurgeCommand.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            } else {
+                deleteBuffer.addAll(toDelete);
+                toDelete.clear();
+            }
+
+            channel.deleteMessages(deleteBuffer).queue();
+            deleteBuffer.clear();
+        }
+
+        channel.sendMessage("Purged the last " + toDeleteSize + " messages from DabBot").queue();
```
Comment (itslukej) [inline src/main/java/ovh/not/javamusicbot/command/PurgeCommand.java]: dabBot*

Comment (Dot-Rar) [reply src/main/java/ovh/not/javamusicbot/command/PurgeCommand.java]: o ok

**Comment by Dot-Rar** [comment]
Done this:

image

**Comment by Dot-Rar** [comment]
very good yes

**Comment by not-john** [comment]
.rar

**Comment by Dot-Rar** [comment]
what cunt

**Comment by ducc** [comment]
sry too many changes to merge at this point, ill stick some of this code into new command system

**Pull request closed without merging**